lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -129,6 +129,7 @@ def test_gridliner_specified_lines():
 grid_label_tol = grid_label_inline_tol = grid_label_inline_usa_tol = 0.5
 if MPL_VERSION >= '2.0':
     grid_label_image = 'gridliner_labels'
+    grid_label_tight_image = 'gridliner_labels_tight'
```

Comment:
This could just be put in the image comparison directly I think since there isn't another version at all.